plugins/fusion/qt.py

```diff
@@ -412,12 +412,15 @@ def user_changed_server(self, *args):
     def update_tor(self,):
         # called on init an switch of auto
         autoport = self.plugin.has_auto_torport()
+        host = self.plugin.get_torhost()
         port = self.plugin.get_torport()
         self.l_tor_status.clear()
         self.torport_update(port)
         self.cb_tor_auto.setChecked(autoport)
+        self.le_tor_host.setEnabled(not autoport)
```

Comment:
Hmm, not sure what is intended here. If user selects 'automatic detection' then it tries to contact whatever host they might have entered before, yet they can't modify it without unchecking then rechecking?

Maybe get_torhost should be changed so that when automatic detection is on, it always returns localhost regardless of the config.

Reply:
That change to `get_torhost` makes sense, I'll do it. I was unsure about the semantics of autodetection. I'll update the code so it means "localhost and autodetect port".